front/src/modules/auth/components/Modal.tsx

```diff
@@ -5,7 +5,7 @@ import { Modal as UIModal } from '@/ui/modal/components/Modal';
 
 type Props = React.ComponentProps<'div'>;
 
-const StyledContainer = styled.div`
+export const StyledContainer = styled.div`
```

Comment:
Auth/Modal should only be used on Auth flow.

Could we instead directly use @/ui/modal/components/Modal instead and add the following props: 
- width: number
- padding: ModalPadding ('Large') (40px)

And add this styledContainer directly in ui/modal